lighthouse-core/report/html/renderer/util.js

```diff
@@ -424,12 +424,9 @@ class Util {
         networkThrottling = Util.i18n.strings.runtimeUnknown;
     }
 
-    let deviceEmulation = Util.i18n.strings.runtimeNoEmulation;
-    if (settings.emulatedFormFactor === 'mobile') {
-      deviceEmulation = Util.i18n.strings.runtimeMobileEmulation;
-    } else if (settings.emulatedFormFactor === 'desktop') {
-      deviceEmulation = Util.i18n.strings.runtimeDesktopEmulation;
-    }
+    const deviceEmulation = settings.formFactor === 'mobile'
```

Comment:
this doesn't seem like the right label here anymore, should we rename this to something more generic?

Reply:
yeah tbh the device-strings bit isn't done. agree this doesnt make sense. 

i was gonna break into a separate PR along with #7053. does that work?

Reply:
heh, absolutely. anything that breaks up what's lurking here sgtm :)

Reply:
https://github.com/GoogleChrome/lighthouse/pull/11796